src/cli/onefuzz/api.py

```diff
@@ -274,6 +274,28 @@ def list(self) -> List[responses.ContainerInfoBase]:
         self.logger.debug("list containers")
         return self._req_model_list("GET", responses.ContainerInfoBase)
 
+    def reset(
+        self, container_types: Optional[List[enums.ContainerType]] = None
+    ) -> None:
+        """Reset all Conatiners unless specified.
+        [Caution]: It can lead to unexpected results.
+        """
+        if not container_types:
+            container_types = enums.ContainerType.__dict__["_member_map_"].values()
```

Comment:
It's easier to just iterate over the enum directoy.  `[x for x in enums.ContainerType]`

However, this erases *all* container types, including those we probably don't want to erase, like the various tools containers (such as `afl-linux`)

Reply:
using `reset_defaults` with suggested iterator.